### ISSUE TITLE
How to setup with own cloudflare credentials

### ISSUE THREAD

**Issue by athersajjad** [subhendukundu/authc1#2]
Can you write a doc on how to setup the development environment?
I tried setting up in my devbox, i saw the wrangler.toml files contains keys that points to your account, doesn't that need to be protected

**Comment by subhendukundu** [comment]
This is something on our roadmap. I will add up an docs to help to start our own authc1 in your Cloudflare account.
To be clear no changes required in the code right now. Just the docs required.

**Comment by AtherSajjad** [comment]
Yes, code changes will not be required, i was trying to setup workers, seems it's not used as of now, and contributing guide will be better
